packages/core/src/codewhispererChat/commands/registerCommands.ts

```diff
@@ -102,6 +110,7 @@ export type EditorContextBaseCommandType =
     | 'aws.amazonq.fixCode'
     | 'aws.amazonq.optimizeCode'
     | 'aws.amazonq.sendToPrompt'
+    | 'aws.amazonq.testCode'
```

Comment:
nit: Can this be named more specific to unit test instead of generalized `testCode`?

Reply:
Yes